akka-docs/rst/project/migration-guide-2.4.x-2.5.x.rst

```diff
@@ -4,6 +4,31 @@
 Upcoming Migration Guide 2.4.x to 2.5.x
 #######################################
 
+Akka Persistence
+================
+
+Removal of PersistentView
+-------------------------
+
+After being deprecated for a long time, and replaced by :ref:`Persistence Query Java <persistence-query-java>`
+(:ref:`Persistence Query Scala <persistence-query-scala>`) ``PersistentView`` has been removed now removed.
+
+The corresponding query type is ``EventsByPersistenceId``. There are several alternatives for connecting the ``Source``
+to an actor corresponding to a previous ``UntypedPersistentView`` actor:
+
+* `Sink.actorRef`_ is simple, but has the disadvantage that there is no back-pressure signal from the
+  destination actor, i.e. if the actor is not consuming the messages fast enough the mailbox of the actor will grow
+* `mapAsync`_ combined with the ``ask`` pattern is almost as simple with the advantage of back-pressure being propagated all the way
+* `ActorSubscriber`_ in case you need more fine grained control
```

Comment:
don't mention `ActorSubscriber`, perhaps it's better to just link to http://doc.akka.io/docs/akka/2.4/scala/stream/stream-integrations.html#Integrating_with_Actors (and corresponding java docs) than describing it here